docs/public-networks/reference/cli/options.md

```diff
@@ -3074,6 +3074,46 @@ Enabling revert reason may use a significant amount of memory. We don't recommen
 
 :::
 
+### `rpc-gas-cap`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--rpc-gas-cap=<INTEGER>
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--rpc-gas-cap=50000000
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_RPC_GAS_CAP=50000000
+```
+
+</TabItem>
+
+<TabItem value="Configuration file" label="Configuration file">
+
+```bash
+rpc-gas-cap=50000000
+```
+
+</TabItem>
+
+</Tabs>
+
+Sets a limit on the amount of gas for transaction simulation RPC methods. Its value must be greater than or equal to `0`. By default, it is set to `0`, which indicates there is no limit. This cap prevents excessive resource use from [eth_call](../api/index.md#eth_call) requests.
```

Comment:
Sets a limit on the amount of gas for transaction simulation RPC methods. Its value must be greater than or equal to `0`. By default, it is set to `0`, which indicates there is no limit. This cap prevents [eth_call](../api/index.md#eth_call) requests from using excessive resources.

Reply:
Added! Thank you